docs/api/connection.rst

```diff
@@ -539,7 +537,7 @@ Pool
         Release a previously acquired connection proxy, to return it to the
         pool.
 
-    .. js:method:: acquireAndExecute<T>(action: func)
+    .. js:method:: run<T>(action: func)
```

Comment:
What's `<T>` here?

Reply:
You're right, I didn't include it in the documentation. In this context, is the same value returned by the `func` that receives a connection.

Reply:
Yeah, I suggest including the full type info here.